src/lib/stories.ts

```diff
@@ -414,15 +415,19 @@ const parseDateComparator: (arg: string) => (date: string) => boolean = arg => {
     };
 };
 
-const checkoutStoryBranch = (story: StoryHydrated, prefix: string = '') => {
+const buildStoryBranch = (story: StoryHydrated, prefix: string = '') => {
     prefix = prefix || `${config.mentionName}/ch${story.id}/${story.story_type}-`;
     let slug = story.name
         .toLowerCase()
         .replace(/\s/g, '-')
```

Comment:
So this line seems to turn a story name of `Upgrade to Rails 5.2` into a branch name of `upgrade-to-rails-52` making it not match the branch name. 

Naive solution would be to add the `.` character like:

`replace(/[\s\.]/g, '-')`

though there may be other cases here where story name doesn't match the hyphenated git integration branch. Are you cool with the above suggestion?

Reply:
Looking at this again this morning I believe we can switch `\s` any whitespace character for `\W` any non-word character. This catches both special characters as well as whitespace

Reply:
I think making the change to use `\W` here (in this pull request) is perfectly fine - I would just make the change in a dedicated commit detailing the reasoning.